extensions/transform/org.eclipse.smarthome.transform.scale/README.md

```diff
@@ -1,16 +1,51 @@
 # Scale Transformation Service
 
-Transform the input by matching it between limits of ranges in a scale file.  The input string must be in numerical format.
+Transform a given input by matching it between limits of ranges.
```

Comment:
I know, you just broke the lines here, but would you agree that "Transform a given input by matching it to ranges" is less confusing?

Reply:
Totally agree.

Wdyt about:
"Transform a given input by matching it to __specific__ ranges."
or
"Transform a given input by matching it to __specified__ ranges."

Just to make directly clear, that there are no fixed ranges somewhere in esh that are used for that.

Reply:
👍  "specified"